packages/katello/nodejs-bootstrap-select/nodejs-bootstrap-select.spec

```diff
@@ -1,18 +1,25 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global npm_name bootstrap-select
-%global enable_tests 1
 
-Name: nodejs-%{npm_name}
+Name: %{?scl_prefix}nodejs-bootstrap-select
 Version: 1.12.4
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Bootstrap-select is a jQuery plugin that utilizes Bootstrap's dropdown
 License: MIT
 Group: Development/Libraries
 URL: http://silviomoreto.github.io/bootstrap-select
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}npm
+%else
 BuildRequires: nodejs-packaging
-Requires: npm(jquery) >= 1.8
+%endif
+Requires: npm(jquery) >= 1.8.0
```

Comment:
This looks incorrect and missing the ``%{?scl_prefix}`. It's repeated in every file so a mistake in the npm2rpm template/code.